### PR TITLE
Menu

### PR DESCRIPTION
**Pull request by Kossecki**
At this stage, the menu is organised in two columns, with each menu item with display flex. There is a space between the two spans of each item.

### CODE REVIEW

File: src/templates/_base.twig [hunk 1]
```diff
@@ -3,6 +3,7 @@
     <head>
         <meta charset="utf-8">
         <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+        <link href="https://fonts.googleapis.com/css?family=Roboto:300&display=swap" rel="stylesheet">
```
Comment (benabraham) [inline src/templates/_base.twig]: all needed variants please (and not this one)

File: src/templates/menu.twig [hunk 2]
```diff
@@ -5,108 +5,132 @@
...
-        <li>Hot chocolate 4.5</li>
-    </ul>
+    <div class="row">
+        <div class="col-sm">
+            <h2>Espresso coffee</h2>
+            <ul class="list-unstyled">
+                <li><div class="d-flex justify-content-between"><span>Espresso</span> <span>3.8</span></div></li>
+                <li><div class="d-flex justify-content-between"><span>Double espresso</span> <span>4.9</span></div></li>
+                <li><div class="d-flex justify-content-between"><span>Americano</span> <span>5</span></div></li>
+                <li><div class="d-flex justify-content-between"><span>Espresso macchiato</span> <span>4.3</span></></li>
```
Comment (benabraham) [inline src/templates/menu.twig]: missing the closing tag here
(validate resulting html please)

Comment (benabraham) [inline src/templates/menu.twig]: divs are not necessary here (same for the rest of the file)

File: src/templates/our-story.twig [hunk 3]
```diff
@@ -6,11 +6,11 @@
 
 {% block content %}
     <h1>Our Story</h1>
-    <img src="img/espresso-machine.jpg" alt="espresso machine close-up" style="height: auto; width: 1090px;" />
+    <img src="img/espresso-machine.jpg" alt="espresso machine close-up" class="img-fluid">
```
Comment (benabraham) [inline src/templates/our-story.twig]: image after text

File: src/templates/menu.twig [hunk 4]
```diff
@@ -5,108 +5,132 @@
...
+        <div class="col-sm">
+            <h2>Sweets</h2>
+            <ul class="list-unstyled">
+                <li><div class="d-flex justify-content-between"><span>Homemade cakes of the day</span> <span>5</span></div></li>
+                <li><div class="d-flex justify-content-between"><span>Key Lime and Coconut</span> <span>1.5</span></div></li>
+                <li><div class="d-flex justify-content-between"><span>Ice cream</span> <span>2</span></div></li>
+                <li><div class="d-flex justify-content-between"><span>Tiramisu</span> <span>6</span></div></li>
+            </ul>
+        </div>
+    </div>
```
Comment (benabraham) [inline src/templates/menu.twig]: try adding sections for main parts of the menu (and corresponding headings if necessary)

**Comment by Kossecki** [comment]
Incorporated suggested changes.

**Pull request merged**